docs/superpowers/plans/2026-08-18-transkribor-wellenplan-3.md

```
@@ -0,0 +1,328 @@
+  Der Abbruch-Merker (#268/#260/#263) ist aus `_lockziel()` **zusammengesetzt**.
+- `:589` — `_lockziel()` ist `settings.path() + ".ytdlp"`. Genau diesen Ausdruck will
+  **#254 Weg 2** ändern. Ändert er sich, ändert sich der Dateiname **jedes** Abbruch-Merkers
+  mit. Eine Migration statt zwei.
+- `:616-632` — der Docstring von `_venv_kennung()` nennt **#254 beim Namen** („dieselbe
+  Zwei-Prozess-Lage wie #254") und hält fest, `electron/backend.js` setze
+  `TRANSKRIBOR_SETTINGS` nicht — **nachgeprüft**. Die halbe Antwort auf #254 steht bereits im
+  Code, mit funktionierendem Präzedenzfall aus PR #259.
+
+**Alle vier Issues beschreiben denselben Satz:** *der Merker sagt etwas anderes, als der
+Zustand ist.* #260: Metadaten heil, Paket halb. #268: Merker abgelaufen, Installation zerlegt.
+#263: Merker liegt, Installation heil. #254 Punkt 2: ein Prozess verbraucht den Tagesmerker
```

markdownlint-cli2 (0.23.2)

[warning] No space after hash on atx style heading

(MD018, no-missing-space-atx)